src/utils/optionsparser.cc

```diff
@@ -67,6 +67,13 @@ void OptionsParser::SetUciOption(const std::string& name,
   throw Exception("Unknown option: " + name);
 }
 
+void OptionsParser::HideOption(const OptionId& id) {
+  auto option = FindOptionByUciName(id.uci_option);
```

Comment:
`FindOptionById(id.GetId())`
This will work even if it's command-line flag only option, without UCI.
(the `GetId()` part will be not needed after future refactorings, and FindOptionById() will get OptionId instead)

Reply:
I saw the FindOptionById prototype and tried it, but it failed to build. I thought it was a placeholder, but maybe it got lost in some refactoring?

Reply:
Yeah, it's `FindOptionById(id.GetId())`, not `FindOptionById(id)`.

Reply:
I mean the implementation of FindOptionById() is missing: 
```
undefined reference to `lczero::OptionsParser::FindOptionById(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
```